arithmetization/src/main/java/net/consensys/linea/zktracer/module/mmio/MmioPatterns.java

```diff
@@ -182,6 +182,23 @@ public static Bytes16 excision(
 
   public static void updateTemporaryTargetRam(
       MmuData mmuData, final long targetLimbOffsetToUpdate, final Bytes16 newLimb) {
+
+    int limbStart = (int) (LLARGE * targetLimbOffsetToUpdate);
```

Comment:
Maybe we could add a simple unit test to validate the method?

Reply:
Agreed, @letypequividelespoubelles, I will need some help on creating a unit test as we don't have one before. I don't visualize yet the real use case.

Reply:
Or maybe @OlivierBBB, as Francois is OOO currently.